Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="MvvmSeed.Android.MvvmSeed.Android" android:versionCode="1" android:versionName="1.0">
+	<uses-sdk />
```

Comment:
Well, it's tricky.. I remember Android complaining about Manifest not having that tag but Xamarin doesn't rely on it.. I'll check and let you know.